packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -336,6 +340,127 @@ describe('', () => {
           'arn:${AWS::Partition}:iam::123456789012:role/cdk-hnb659fds-cfn-exec-role-123456789012-us-west-2');
 
 
+      });
+      test('generates the same support stack containing the replication Bucket without the need to bootstrap in that environment for multiple pipelines', () => {
+        const app = new cdk.App({
+          treeMetadata: false, // we can't set the context otherwise, because App will have a child
+        });
+        app.node.setContext(cxapi.NEW_STYLE_STACK_SYNTHESIS_CONTEXT, true);
```

Comment:
Why are we setting this flag...?

Reply:
I had copied it from a test above, and didn't notice it was there. I have removed it.